src/features/auth/components/WithoutAuth.tsx

```diff
@@ -0,0 +1,17 @@
+'use client';
+
+import { useAuth } from '@processes/auth';
+import { PropsWithChildren, useEffect, useState } from 'react';
+
+function WithoutAuth({ children }: PropsWithChildren) {
+  const { authorized } = useAuth();
+  const [isVisible, setIsVisible] = useState(true);
+
+  useEffect(() => {
+    setIsVisible(!authorized);
+  }, [authorized]);
+
+  return <>{isVisible ? children : null}</>;
```

Comment:
Please check the that I left for `WithAuth` component, I think that we can slightly improve readability here